debugger/src/main/java/org/codice/acdebugger/breakpoints/SecurityCheckInformation.java

```diff
@@ -361,26 +223,29 @@ public void dump(boolean osgi, String prefix) {
                 .mapToObj(i -> "=")
                 .collect(Collectors.joining("")));
     dump0(osgi);
-    if (!isAcceptable()) {
-      for (int i = 0; i < analysis.size(); i++) {
-        final SecuritySolution info = analysis.get(i);
+    for (int i = 0; i < analysis.size(); i++) {
```

Comment:
ℹ️ By design now, no solutions are returned if it is an acceptable failures. As such, this method would no longer be called as it is part of the solution api and not the failure api